website/pages/en/developing/graph-ts/api.mdx

```diff
@@ -29,6 +29,8 @@ The `apiVersion` in the subgraph manifest specifies the mapping API version whic
 
 | Version | Release notes |
 | :-: | --- |
+| 0.0.9 | Adds new host function `eth_get_balance` |
```

Comment:
apiVersion 0.0.9 also int

Reply:
I think that is the `specVersion` (which is about the structure of the manifest) rather than the `apiVersion`?